### PR TITLE
RFC: V2 paths

### PR DESCRIPTION
**Pull request by sunnygleason**
@mvines : FYI, rough idea for /v2/ coexisting with Blockexplorer V1
@CriesofCarrots : courtesy/honorary review FYI!

Problem:

* need to do v2 development while keeping v1 running

Proposed Solution:

* add a v2 render path that is executed when the path matches '/v2/*'
* start naming convention of Bx2* for Blockexplorer v2 components

Open Questions:

* Can tachyonsCSS play nice with react material-ui components?
* Or, is tachyonsCSS sufficient to all-out replace use of material-ui?

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -724,6 +737,71 @@ class App extends Component {
...
     );
   }
+
+  renderV2() {
+    let self = this;
+
+    const leaderId = this.state.globalStats['!ent-last-leader'];
+
+    //
+    // open questions
```
Comment (mvines) [inline src/App.js]: Let's keep this kind of stuff out of the code.  Use github issues, etc to track

**Comment by mvines** [comment]
Should we create `src/v2/` for all the v2 stuff?  `src/App.js` then redirects to `src/v2/App.js` for v2 routes.  I don't have a super strong opinion about this, just seems like a nice way to keep the old/new separate

**Comment by sunnygleason** [comment]
> Should we create `src/v2/` for all the v2 stuff? `src/App.js` then redirects to `src/v2/App.js` for v2 routes. I don't have a super strong opinion about this, just seems like a nice way to keep the old/new separate

I like that idea! let's see what Devon thinks...

File: src/App.js [hunk 2]
```diff
@@ -234,7 +247,10 @@ class App extends Component {
     );
 
     try {
-      const [/*blockhash*/, feeCalculator] = await this.connection.getRecentBlockhash();
+      const [
```
Comment (sunnygleason) [inline src/App.js]: @mvines this is an unanticipated lint change, does it bother you?

Comment (mvines) [reply src/App.js]: meh oh well :)

**Comment by mvines** [comment]
> > Should we create `src/v2/` for all the v2 stuff? `src/App.js` then redirects to `src/v2/App.js` for v2 routes. I don't have a super strong opinion about this, just seems like a nice way to keep the old/new separate
> 
> I like that idea! let's see what Devon thinks...

Just do it

**Comment by sunnygleason** [comment]
@mvines I think this is a good first pass at how the style/theme is split out. I'll refine more tomorrow. In the meantime, do you think this is ok to merge?

**Comment by mvines** [comment]
Yep, :shipit:

**Pull request merged**